packages/zoe/src/cleanProposal.js

```diff
@@ -29,7 +29,11 @@ export const assertKeywordName = keyword => {
   assert.typeof(keyword, 'string');
   keyword.length <= MAX_KEYWORD_LENGTH ||
     assert.fail(
-      X`keyword ${keyword} exceeded maximum length ${MAX_KEYWORD_LENGTH} characters; got ${keyword.length}`,
+      X`keyword ${q(
+        keyword,
+      )} exceeded maximum length ${MAX_KEYWORD_LENGTH} characters; got ${
+        keyword.length
```

Comment:
Likely my mistake, but as long as you're in the immediate vicinity, might as well fix it too. No harm quoting both `MAX_KEYWORD_LENGTH` and `keyword.length`.

Reply:
I pushed a fix and neglected to quote `keyword.length`. I'm inclined to push as is because the actual keyword is there to observe. (if I push again I think I'd just remove the `; got…` part.

PTAAL at the other change in [4a21b80](https://github.com/Agoric/agoric-sdk/pull/6261/commits/4a21b8076fd08edd39c70cb1929a0b0921336aa3)